client/src/components/Result/Result.jsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Tooltip } from "@mui/material";
+import {
+  bestPossibleAverage,
+  worstPossibleAverage,
+  formatAttemptResult,
+  incompleteMean,
+} from "../../lib/attempt-result";
+
+const Result = ({ result, field, eventId, roundFormat }) => {
+  return (
+    <>
+      {result.average === 0 && field === "average"
+        ? roundFormat.id === "a"
+          ? result.attempts.length > 3 && (
```

Comment:
Now that it's a separate component, we can use regular `if`/`else` to improve readability, something along these lines:

```jsx
if (result.average === 0 && field === "average") {
  if (roundFormat.id === "a") {
    return ...
  }

  ...
}

return ...
```